mfr/extensions/tabular/libs/stdlib_tools.py

```diff
@@ -1,57 +1,91 @@
 import re
 import csv
+from http import HTTPStatus
 
-from mfr.extensions.tabular.exceptions import EmptyTableError, TabularRendererError
 from mfr.extensions.tabular import utilities
+from mfr.extensions.tabular.exceptions import (EmptyTableError,
+                                               TabularRendererError)
 
 
 def csv_stdlib(fp):
-    """Read and convert a csv file to JSON format using the python standard library
-    :param fp: File pointer object
-    :return: tuple of table headers and data
-    """
-    data = fp.read(2048)
+    data = fp.seek(2048)
     fp.seek(0)
+    # set the dialect instead of sniffing for it.
+    # sniffing can cause things like spaces or characters to be the delimiter
+    dialect = csv.excel
```

Comment:
Hmmm.  I'm not sure how I feel about this.  I like that it solves the issue of really long lines, but it bugs me a bit that we're throwing out support for alternative delimiters.  If we use the `csv.excel` dialect, do we still support tab- and pipe-delimited text?  If not, can we document that in a comment, so we'll know what to fix if we encounter it?

Reply:
Complete. [PR308](https://github.com/CenterForOpenScience/modular-file-renderer/pull/308)